### DISCUSSION TITLE
How to upload the file over API

### DISCUSSION THREAD

**Discussion by Jianbin719** [YetiForceCompany/YetiForceCRM#17592] [How to] [answered]
I can create ticket with API, but the problem is that I need image or attachment both for creating ticket.  The files were failed to send over api. 

Is the way same? or there is any trick??

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
see here

https://doc.yetiforce.com/developer-guides/api/how-to-add-graphic-file

https://doc.yetiforce.com/developer-guides/api/how-to-add-document-with-an-attachment/

**Comment by Jianbin719** [comment]
Noted with thanks.
